packages/commerce-sdk-react/src/test-utils.tsx

```diff
@@ -86,3 +87,50 @@ export function renderHookWithProviders<TProps, TResult>(
         )
     })
 }
+
+export const NEW_DATA = {test: 'new data'}
+export const OLD_DATA = {test: 'old data'}
+
+export const mockMutationEndpoints = (matchingPath: string, options?: {errorResponse: number}) => {
+    const responseStatus = options?.errorResponse ? options.errorResponse : 200
+
+    nock(DEFAULT_TEST_HOST)
+        .patch((uri) => {
+            return uri.includes(matchingPath)
+        })
+        .reply(responseStatus, NEW_DATA)
+        .put((uri) => {
+            return uri.includes(matchingPath)
+        })
+        .reply(responseStatus, NEW_DATA)
+        .post((uri) => {
+            return uri.includes(matchingPath)
+        })
+        .reply(responseStatus, NEW_DATA)
+        .delete((uri) => {
+            return uri.includes(matchingPath)
+        })
+        .reply(options?.errorResponse ? options.errorResponse : 200, NEW_DATA)
```

Comment:
We can simplify it like this (because the same logic is already at line 95 above):
```suggestion
        .reply(responseStatus, NEW_DATA)
```